### PR TITLE
Ledger: Make messages refer to the device by name and enable localization

### PR DESCRIPTION
**Pull request by EchterAgo**
From Telegram:

> Any idea what error “dongle is temporarily locked - please unplug it and replug it again” means?

Users can be easily confused by just saying dongle or device so we should refer to devices by their names. This also makes more messages available for localization.

### CODE REVIEW

File: plugins/ledger/ledger.py [hunk 1]
```diff
@@ -174,26 +175,26 @@ def perform_hw1_preflight(self):
             if self.has_detached_pin_support(self.dongleObject) and not self.is_pin_validated(self.dongleObject) and (self.handler is not None):
                 remaining_attempts = self.dongleObject.getVerifyPinRemainingAttempts()
                 if remaining_attempts != 1:
-                    msg = "Enter your Ledger PIN - remaining attempts : " + str(remaining_attempts)
+                    msg = _('Enter your {} PIN - remaining attempts: {}').format(self.device, remaining_attempts)
                 else:
-                    msg = "Enter your Ledger PIN - WARNING : LAST ATTEMPT. If the PIN is not correct, the dongle will be wiped."
+                    msg = _('Enter your {} PIN - WARNING: LAST ATTEMPT. If the PIN is not correct, the dongle will be wiped.').format(self.device)
```
Comment (cculianu) [inline plugins/ledger/ledger.py]: You missed a "dongle" here on like 180?  ;)

**Comment by cculianu** [comment]
Uh. "Dongle" is a nasty word.  Good work here.  Consistency is key.

You did miss a "dongle" though (see comments). Heh.

File: plugins/ledger/ledger.py (hunk 1 above)
Comment (EchterAgo) [reply plugins/ledger/ledger.py]: Ah, thanks, will push an update.

**Comment by EchterAgo** [comment]
Updated

**Comment by cculianu** [comment]
OK, merging.

**Pull request merged**